ksql-engine/src/main/java/io/confluent/ksql/physical/PhysicalPlanBuilder.java

```diff
@@ -214,6 +234,23 @@ private String addTimeSuffix(String original) {
     return String.format("%s_%d", original, System.currentTimeMillis());
   }
 
+  private void updateListProperty(Map<String, Object> properties, String key, Object value) {
+    Object obj = properties.getOrDefault(key, new java.util.LinkedList<String>());
```

Comment:
I guess there no need to import LL with full path. `LinkedList` is enough as it already imported.